test/cmocka/src/math/arithmetic/base2_logarithm.c

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2021 Intel Corporation. All rights reserved.
+//
+// Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+
+#include <stdio.h>
+#include <stdint.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <math.h>
+#include <cmocka.h>
+
+#include <sof/audio/format.h>
+#include <sof/math/log.h>
+#include <sof/common.h>
+#include "log2_tables.h"
+/* 'Error[max] = 0.0000236785999981,THD(-dBc) = -92.5128795787487235' */
+#define CMP_TOLERANCE 0.0000236785999981
+
+static void test_math_arithmetic_base2log_fixed(void **state)
+{
+	(void)state;
+
+	double u;
+	double v;
+	int b_i;
+	int i;
+
+	for (i = 0; i < ARRAY_SIZE(log2_lookup_table); i++) {
+		u = ((double)vector_table[i] / (1 << 1));
```

Comment:
`1 << 1` can just be `2`?

Reply:
Yes that correct . `1<<1` can be done 2.